build-challenge/just.config.ts

```diff
@@ -0,0 +1,89 @@
+import { argv, parallel, series, task, tscTask } from "just-scripts";
+import {
+  BundleTaskParams,
+  CleanCollateralTaskParams,
+  CopyTaskParameters,
+  bundleTask,
+  cleanTask,
+  cleanCollateralTask,
+  copyTask,
+  coreLint,
+  generateContentsJsonTask,
+  mcaddonTask,
+  setupEnvironment,
+  ZipTaskParameters,
+  GenerateContentsJsonParameters,
+  updateWorldTask,
+} from "@minecraft/core-build-tasks";
+import path from "path";
+
+const buildTaskOptions: BundleTaskParams = {
+  entryPoint: path.join(__dirname, "./scripts/main.ts"),
+  external: ["@minecraft/server", "@minecraft/server-ui"],
+  outfile: path.resolve(__dirname, "./dist/scripts/main.js"),
+  minifyWhitespace: false,
+  sourcemap: true,
+};
+
+const cleanTaskOptions: CleanCollateralTaskParams = {
+  pathsToClean: [
+    "LOCALAPPDATA/Packages/Microsoft.MinecraftUWP_8wekyb3d8bbwe/LocalState/games/com.mojang/development_behavior_packs/PROJECT_NAME",
+    "LOCALAPPDATA/Packages/Microsoft.MinecraftUWP_8wekyb3d8bbwe/LocalState/games/com.mojang/development_resource_packs/PROJECT_NAME",
+    "LOCALAPPDATA/Packages/Microsoft.MinecraftWindowsBeta_8wekyb3d8bbwe/LocalState/games/com.mojang/development_behavior_packs/PROJECT_NAME",
+    "LOCALAPPDATA/Packages/Microsoft.MinecraftWindowsBeta_8wekyb3d8bbwe/LocalState/games/com.mojang/development_resource_packs/PROJECT_NAME",
+  ],
+};
+
+const generateBehaviorPackContentsJsonOptions: GenerateContentsJsonParameters = {
+  targetPath: "./behavior_packs/buildchallenge",
+  outputFile: "./dist/behavior_pack/contents.json",
+};
+
+const generateResourcePackContentsJsonOptions: GenerateContentsJsonParameters = {
+  targetPath: "./resource_packs/buildchallrp",
+  outputFile: "./dist/resource_pack/contents.json",
+};
+
+const copyTaskOptions: CopyTaskParameters = {
+  copyToBehaviorPacks: ["./behavior_packs/buildchallenge", "./dist/behavior_pack/contents.json"],
```

Comment:
`buildchallenge`? Is that the PROJECT_NAME, doesn't the task infer that?

Reply:
This is the local path (inside the project folder) of the behavior pack, the target path is inferred as you say.
If we want to support multiple BP or RP in the future, we should support any input path

Reply:
Using the env variable instead, now files are same, except build challenge that packs a world